compiler/semcall.nim

```diff
@@ -212,7 +210,6 @@ proc resolveOverloads(c: PContext, n, orig: PNode,
       if result.state != csMatch:
         n.sons.delete(1)
         orig.sons.delete(1)
-        excl n.flags, nfExprCall
```

Comment:
There is nothing wrong with GitHub, Araq has reviewed each commit individually so this diff is for one of your commits.